src/plugins/vis_type_markdown/kibana.json

```diff
@@ -0,0 +1,6 @@
+{
+    "id": "markdown_vis",
```

Comment:
@flash1293 I wonder if we change ids to camel case during the migration?

Reply:
Yes, we should do this and to make everything consistent, we should also rename the plugin to the same naming scheme we use with the folders: `visTypeMarkdown`.

Also the same things as in the metric vis type PR apply: https://github.com/elastic/kibana/pull/63096#pullrequestreview-392872060